docs/memory-usage-reduce-1160-1180.md

```diff
@@ -0,0 +1,34 @@
+# Config Sync Memory Usage Reduction v1.16 vs v1.18
+
+Config Sync v1.18 contains change to stop loading OpenAPI for schema validations when Config Sync admission webhook is not enabled. Here are some test results that shows the reduction in the memory usage.
```

Comment:
Can you explain the problem first?

How does using OpenAPI increase the memory? In what use cases?

Reply:
@nan-yu did profiling previously and loading the OpenAPI consumes a sizable fraction of the memory.